src/app/components/mayastor/mayastor.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="container-fluid bg-white py-2">
+    <h5>
+        Mayastor ci pipelines
```

Comment:
Can we use translation for these strings ? Here and other places as well, instead of raw strings it is better if we could convert them to translated strings.